src/tests/issues.targets

```diff
@@ -2988,10 +2988,13 @@
         <ExcludeList Include="$(XunitTestBinBase)/baseservices/varargs/varargsupport_r/*">
             <Issue>https://github.com/dotnet/runtime/issues/57350</Issue>
         </ExcludeList>
-        <ExcludeList Include = "$(XunitTestBinBase)/baseservices/TieredCompilation/TieredVtableMethodTests/**">
+        <ExcludeList Include="$(XunitTestBinBase)/baseservices/TieredCompilation/TieredVtableMethodTests/**">
+            <Issue>https://github.com/dotnet/runtime/issues/57350</Issue>
+        </ExcludeList>
+        <ExcludeList Include="$(XunitTestBinBase)/Loader/classloader/explicitlayout/objrefandnonobjrefoverlap/case2/**">
             <Issue>https://github.com/dotnet/runtime/issues/57350</Issue>
         </ExcludeList>
-        <ExcludeList Include = "$(XunitTestBinBase)/Loader/classloader/explicitlayout/objrefandnonobjrefoverlap/case2/**">
+        <ExcludeList Include="$(XunitTestBinBase)/Loader/classloader/explicitlayout/objrefandnonobjrefoverlap/case13/**">
```

Comment:
I'm surprised this is necessary: Directory.Build.props for the whole `objrefandnonobjrefoverlap` directory specifies `<MonoAotIncompatible>true</MonoAotIncompatible>` which should skip the test.

That said, I think #57350 is masking a real issue here; arguably `MonoAotIncompatible` is wrong here anyway - the test should AOT and pass.

Reply:
I see: I misunderstood the intent of `MonoAotIncompatible`.  It only blocks AOT compilation.  But it doesn't prevent the test from attempting to run. Hence we need to skip it here in the FullAOT issues.targets section.

Reply:
@jkoritzinsky Just to confirm the scenario above. The MonoAOT scenarios require two blockers, right? The first is for the AOT step and the second, in issues.targets, is for running that particular test.

Reply:
Also maybe the right thing for case13 (which is a positive test) is to **un**set `MonoAotIncompatible` overriding what's in Directories.Build.props; and not to add to issues.targets.  This test should pass.  Evidently it passes on the Mono non-AOT scenarios - so we don't throw an incorrect loader exception or anything like that.  It's just that we skipped AOTing it, and now the AOT-only runtime is complaining that it's being asked to JIT a test case.

Reply:
Yes, you need to add two blocks. The MonoAotIncompatible blocks the AOT step and the issues.targets blocks the execution.

Reply:
@lambdageek Sure. I can give that a shot.